CONTRIBUTING.md

```diff
@@ -66,9 +69,10 @@ There are many ways to get in touch with the Turing Way team!
     Open a new issue!
     (See our [Where to start: issues](#where-to-start-issues) section below.)
 - Subscribe to our [mailing list](https://tinyletter.com/TuringWay) with which we send monthly project updates.
-- Check out the [#TuringWay](https://twitter.com/search?q=%23TuringWay&src=typed_query) hashtag on Twitter.
-- You can contact the PI of the Turing Way project - Kirstie Whitaker - by email at kwhitaker@turing.ac.uk.
-- You can also contact members of the Turing Way team through their preferred ways of communication [here](https://github.com/alan-turing-institute/the-turing-way/blob/master/ways_of_working.md).
+- Follow [@turingway on Twitter](https://twitter.com/turingway).
+- You can contact the Project Lead of the Turing Way project - Kirstie Whitaker - by email at kwhitaker@turing.ac.uk.
+- You can contact the Community Manager of the Turing Way project - Malvika Sharan - by email at masharan@turing.ac.uk.
+- You can also contact other members of the Turing Way through their preferred ways of communication [here](https://github.com/alan-turing-institute/the-turing-way/blob/master/ways_of_working.md).
```

Comment:
What do you think about removing this? I feel like it is so out of date that it probably needs to be archived now...

Reply:
I think it really depends on the preference of other members who have strong ties with the project and still active contributors.

Reply:
Lets leave it for now and maybe come back to it in the future :)